proposals/dao/tip_121b.ts

```diff
@@ -149,6 +162,87 @@ const validate: ValidateUpgradeFunc = async (addresses, oldContracts, contracts,
   await merkleRedeemerDripper.drip();
   const redeemerBalAfterDrip = await fei.balanceOf(rariMerkleRedeemer.address);
   expect(redeemerBalAfterDrip.sub(redeemerBalBeforeDrip)).to.be.equal(dripAmount);
+
+  await logDaiBalances(contracts.dai);
+
+  // Execute fuseWithdrawalGuard actions
+  let i = 0;
```

Comment:
Why are we doing this fuse withdrawal in the `validate()` fn?  What's the need?

Reply:
Just to make sure the new parameters are working